taurus/entity/object/process_spec.py

```diff
@@ -82,3 +84,17 @@ def _unset_ingredient(self, ingred):
     def output_material(self):
         """Get the output material spec."""
         return self._output_material
+
+    @property
+    def _forward(self) -> Iterable[str]:
+        """List of forward links that should be skipped when doing a chronological traversal.
+
+        Usually, this is the same as the skip list.  However, the link direction between
```

Comment:
Referencing the documentation of `BaseEntity._forward` here, rather than duplicating it, might reduce the future cost of avoiding diverging documentation.

Reply:
Given that we're currently spending most of our time in-code rather than in sphinx docs, I think this duplication is warranted.